src/app/api/v1/applications/[appSlug]/tickets/[id]/distribute/route.ts

```diff
@@ -65,6 +68,19 @@ async function postHandler(req: NextRequestWithDeveloperUserAccessToken & NextRe
       contractArtifacts["tickets"].abi,
     );
 
+    const emailsToSend = await Promise.all(
+      validBody.data.distributions.map(async (receiver: any) => ({
+        recipientEmail: receiver.email,
+        subject: `Your ticket to adicon!`,
+        html: await renderTicketReceiverEmail({
+          eventName: "adicon",
+          ticketUrl: "",
+          imageUrl: ""
+        })
+      }))
+    );
+    const emailResults = await sendBatchEmails(emailsToSend, req.nextUrl.hostname === "localhost");
```

Comment:
new way of sending mulitple emails in paralel (can be dangerous if we hit some limit?)